src/openforms/formio/api/validators.py

```diff
@@ -158,3 +158,9 @@ def __call__(self, uploaded_file: UploadedFile) -> None:
                 raise serializers.ValidationError(
                     _("The virus scan returned an unexpected status.")
                 )
+
+
+class TruthyBooleanValueValidator:
+    def __call__(self, value):
+        if not value:
+            raise serializers.ValidationError(_("This field must be true."))
```

Comment:
It is required in my opinion since we want to make sure we have `True` value when it should be checked (I was only passing `required=required` which doesn't have anything to do with the real value), correct?
And I had a mistake in the tests, I hadn't provided the right value in order to trigger the validator.

Reply:
I think this is already handled through the `required` for a `serializers.BooleanField`, that would explain why this code is not covered.

Reply:
But the test is failing without this validator and by only having `serializers.BooleanField(required=required)`. Am I missing something else?I thought that the `required` is only for the field to be there but not the explicit boolean value.

Reply:
I will have to look into it, because the missing coverage seems to indicate this validator isn't running at all :thinking: